raiden-cli/src/routes/pendingTransfers.ts

```diff
@@ -1,20 +1,61 @@
 import { Router, Request, Response } from 'express';
+import { timer } from 'rxjs';
+import { filter, takeUntil, map, toArray } from 'rxjs/operators';
+import { RaidenTransfer, isntNil } from 'raiden-ts';
+
 import { Cli } from '../types';
+import { validateOptionalAddressParameter } from '../utils/validation';
 
-export function makePendingTransfersRouter(this: Cli): Router {
-  const router = Router();
+enum ApiRole {
+  initiator = 'initiator',
+  mediator = 'mediator',
+  target = 'target,',
+}
 
-  router.get('/', function (_request: Request, response: Response): void {
-    response.status(404).send('Not implemented yet');
-  });
+function transformSdkToApiPendingTransfer(this: Cli, transfer: RaidenTransfer) {
```

Comment:
Similar to the comment above. So far these conversations where exported into a separate module. The intention was to keep the routes slim and focused on retrieving data from the SDK or trigger events. Such details just clutter it. 
But I actually don't have too hard feelings about that. Just that we should attempt to develop a uniform code-base. So again: either do it like we do for the other resources already or adjust all other resources.

Reply:
Such details are utility functions only useful to these local routes, not relevant to anyone else, so I don't think it gets cluttered by them. We should decide on this and do this refactoring on a separate PR.